Kitodo-LongTimePreservationValidationModule/src/main/resources/jhove.conf

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<jhoveConfig version="1.0"
```

Comment:
If this file should be part of a normal Kitodo.Production configuration environment then Kitodo license header is needed.